.github/pull_request_template.md

```diff
@@ -2,7 +2,15 @@
 
 
 ## Code Review Checklist
- * Does the PR describe what changes are being made?
- * Does the code follow the [Effective Go style guide](https://golang.org/doc/effective_go.html)?
- * Does the new code have enough documentation? (comments and or diagrams)
- * Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
+ - [ ] Does the PR describe what changes are being made?
+ - [ ] Does the PR describe why the changes are being made?
+ - [ ] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
+ - [ ] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
+ - [ ] Could the PR be broken into smaller PRs?
+ - [ ] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
+ - [ ] Does the new code have enough documentation? (comments and or diagrams)
+ - [ ] Does any documentation need updating?
+
+Thanks for submitting a PR!
+
+![](https://media.giphy.com/media/14nakW0jC4HA6k/giphy.gif)
```

Comment:
this gif isn't rendering for me when i "view" this file 😢

Reply:
i thiiiiink it should work in the actual PR though. i think it's a bit more loose